### PR TITLE
Update citing aspect

### PR DESCRIPTION
**Pull request by ljhwang**
Attempt to clean-up manual.bib

- aligns manual bibtex type with citing_aspect.bib
- resolves many preprint, submitted, and ??? entries. Not all entries were resolvable but it is a start!

File was checked in separate parser. However References should be check the next time the manual is compiled to ensure this was done correctly and no new errors were introduced.

### CODE REVIEW

File: doc/manual/manual.bib [hunk 1]
```diff
@@ -423,15 +423,13 @@ @Article{BSTOL02
...
-  OPTmonth =     {},
-  OPTnote =      {},
-  OPTannote =    {}
+  year = {2002},
+  month = {06},
+  pages = {373-388},
+  volume = {199},
+  journal = {Earth and Planetary Science Letters},
+  doi = {10.1016/S0012-821X(02)00597-6}
+                  }
```
Comment (tjhei) [inline doc/manual/manual.bib:432]: is this } correct here?

File: doc/manual/manual.bib [hunk 2]
```diff
@@ -2684,6 +2691,7 @@ @Book{Lev92
   title =        {Numerical Methods for Conservation Laws},
   publisher =    {Birkh{\"a}user},
   year =         {1992},
+  doi =          https://doi.org/10.1007/978-3-0348-5116-9,
```
Comment (tjhei) [inline doc/manual/manual.bib:2694]: can you put {} around please?

File: doc/manual/manual.bib [hunk 3]
```diff
@@ -3981,29 +3997,35 @@ @Article{CKPS01
...
-  title =        {Newton Type Methods for the Approximate Solution
-                  of Nonlinear Programming Problems in  Real-Time},
-  institution = {University of Heidelberg},
-  year =         2001,
-  note = {to appear}
+  editor="Di Pillo, G. and Murli, A.",
+  title="Newton-Type Methods for the Approximate Solution of Nonlinear Programming Problems in Real-Time",
+  bookTitle="High Performance Algorithms and Software for Nonlinear Optimization",
+  year="2003",
+  publisher="Springer US",
```
Comment (tjhei) [inline doc/manual/manual.bib:4022]: would you mind changing the quotes here to {} for all lines?

File: doc/manual/manual.bib [hunk 4]
```diff
@@ -4022,7 +4044,9 @@ @TechReport{Die01
   institution =  {University of Heidelberg},
   year =         2001,
   number =       {SFB Preprint 2001-07},
-  note =         {Ph.D. Thesis}
+  note =         {Ph.D. Thesis},
```
Comment (tjhei) [inline doc/manual/manual.bib:4047]: this should probably be of type ``@phdthesis``?

File: doc/manual/manual.bib [hunk 5]
```diff
@@ -4022,7 +4044,9 @@ @TechReport{Die01
...
+  url =         http://www.ub.uni-heidelberg.de/archiv/1431,
```
Comment (tjhei) [inline doc/manual/manual.bib:4048]: add {}

File: doc/manual/manual.bib [hunk 6]
```diff
@@ -4087,14 +4111,10 @@ @Article{GK
...
-  OPTvolume =    {},
-  OPTnumber =    {},
-  OPTpages =     {},
-  OPTmonth =     {},
-  OPTnote =      {},
-  OPTannote =    {}
+  year =         {2003},
+  volume =       95,
+  pages =        {527-550},
+  url = https://doi.org/10.1007/s002110200392
```
Comment (tjhei) [inline doc/manual/manual.bib:4117]: same here

File: doc/manual/manual.bib [hunk 7]
```diff
@@ -4186,16 +4212,25 @@ @Article{HR00a
...
-  author =       {Heuveline, V. and Rannacher, R.},
-  title =        {A numerical tool for flow simulation in a {W}ankel motor},
-  booktitle =    {Mathematik: Schl{\"u}sseltechnologie f{\"u}r die Zukunft},
-  publisher =    {Springer, Berlin-Heidelberg},
-  year =         2000,
-  editor =       {Hoffmann, K. -H. et al.},
-  note =         {to appear}
+
+@Inbook{HR00b,
+  author="Rannacher, Rolf
```
Comment (tjhei) [inline doc/manual/manual.bib:4217]: put on one line

File: doc/manual/manual.bib [hunk 8]
```diff
@@ -4186,16 +4212,25 @@ @Article{HR00a
...
+  and Heuveline, Vincent",
+  editor="J{\"a}ger, Willi
+  and Krebs, Hans-Joachim",
+  title = "A Numerical Tool for Flow Simulation in a Wankel Motor",
```
Comment (tjhei) [inline doc/manual/manual.bib:4221]: again, I would prefer {} over ""

File: doc/manual/manual.bib [hunk 9]
```diff
@@ -10601,27 +10682,30 @@ @MastersThesis{Sch95
...
-  year =         1997,
-  type =         {Preprint SC 97-64}
+  year =         1998,
+  volume =      30,
+  pages =       427--4641,
+  doi =         https://doi.org/10.1023/A:1006919107753
 }
 
-
+-441
```
Comment (tjhei) [inline doc/manual/manual.bib:10708]: what is this?

Comment (ljhwang) [reply doc/manual/manual.bib:10708]: artifact. i know who i am calling the next time I am looking for a needle in a haystack.

File: doc/manual/manual.bib (hunk 8 above)
Comment (ljhwang) [reply doc/manual/manual.bib:4221]: These come free with some citation exporters.

File: doc/manual/manual.bib (hunk 1 above)
Comment (ljhwang) [reply doc/manual/manual.bib:432]: Nice catch. Interesting that my parser did not catch that!

**Comment by ljhwang** [comment]
a6bd93b fixes all legacy and introduced "" 's replacing them with the preferred {}'s.
And addresses other misc comments above.

File: doc/manual/manual.bib (hunk 8 above)
Comment (bangerth) [reply doc/manual/manual.bib]: That's an example of a publication not worth updating, given that it clearly has no relevance to us...

**Comment by tjhei** [comment]
the documentation currently doesn't compile with your changes:
```
Database file #1: manual.bib
I was expecting a `,' or a `}'---line 10701 of file manual.bib
 :   pages =       427
 :                    --4641,
I'm skipping whatever remains of this entry
I was expecting a `,' or a `}'---line 10887 of file manual.bib
 :     title      = {Trilinos} 
 :                             web page},
```
Can you please take a look?

**Comment by ljhwang** [comment]
Thanks for pointing these out.

**Comment by ljhwang** [comment]
This addresses a few formatting errors notice by Timo and closes #3696

File: doc/manual/manual.bib (hunk 2 above)
Comment (tjhei) [reply doc/manual/manual.bib:2685]: I think this still needs {}

**Comment by ljhwang** [comment]
I suspect in a legacy file of 11,863 lines there are many of these. Sorry I missed this in your initial comments.

**Comment by bangerth** [comment]
I don't think that testing would do any good here, so I'll take a leap of faith and merge it!

**Pull request merged**